source/geometry/FloatBox2D.ooc

```diff
@@ -122,7 +122,7 @@ FloatBox2D: cover {
 		newSize := FloatVector2D linearInterpolation(this size, other size, weight)
 		This createAround(newCenter, newSize)
 	}
-	toString: func -> String { "#{this leftTop toString()}, #{this size toString()}" }
+	toString: func -> String { this leftTop toString() & ", " clone() & this size toString() }
```

Comment:
Maybe `(this leftTop toString() >> ", ") & this size toString()` instead of `clone` ?